src/sagemaker/workflow/utilities.py

```diff
@@ -151,3 +158,41 @@ def validate_step_args_input(
         raise TypeError(error_message)
     if step_args.caller_name not in expected_caller:
         raise ValueError(error_message)
+
+
+def override_pipeline_parameter_var(func):
+    """A temporary Decorator
```

Comment:
we might need this in the future too, but we can add it back anytime when we need it.

Reply:
Yea... I'm thinking about maybe we can keep this annotation definition here but removing the annotation on top of `image_uris.retrieve` once users are ready to accept the breaking change